Editor/Window/ElementLocalizer.cs

```diff
@@ -29,9 +29,10 @@ public void SetElementText(string elementName, string textKey)
         }
         
         // The status box use case where you would want to display an Exception message in addition to the generic error text.
-        public void SetElementText(string elementName, string textKey, string additionalText)
+        public void SetElementText(string elementName, string textKey, string additionalTextKey)
         {
             var text = _textProvider.Get(textKey);
+            var additionalText = _textProvider.Get(additionalTextKey);
```

Comment:
The purpose of `additionalText` was to append an exception message, which is directly passed down by the API response. This change will break those cases since that message will not be present within the TextProvider. 

Also, `Invalid details` provides no context for users to fix the issue they are facing. What details are invalid? Instead of modifying this method I would suggest passing `_textProvider.Get(additionalTextKey);` to the additionalText parameter instead. Please use a more verbose and accurate error message over `Invalid details`.

Reply:
Will look at this on monday, could you provide an improved message? The context is that they are missing details in 1 or more of the fields for creating a profile.

Reply:
Which flow is this coming through? I know we disable the UX in all cases where there is missing information, and wondering if that's a possibility? I don't recall if we allow clicking Submit in any other use case where there is missing information, could you confirm? 

If not, "The <field> cannot be empty" seems like a good start

Reply:
I have now changed it from a status box to just not allowing the user to click create profile without having all of the fields and dropdown filled out. Removes the need for addition text in this context.

Reply:
The additionalText was meant to be a way to pass down error messages returned by the AWS SDK, which doesn't need translation. The reason this still works is the TextProvider returns the key if it can't find it in the map. 

I think this is ok, since this is generic enough that it works for someone wanting to pass down a key as well. a